auth/src/main/java/org/aerogear/mobile/auth/configuration/AuthServiceConfiguration.java

```diff
@@ -57,6 +65,11 @@ public AuthConfigurationBuilder allowSelfSignedCertificate(boolean allowSelfSign
             return this;
         }
 
+        public AuthConfigurationBuilder withMinTimeBetweenJwksRequests(int minTimeBetweenJwksRequests) {
+            this.minTimeBetweenJwksRequests = minTimeBetweenJwksRequests;
+            return this;
+        }
+
         public AuthServiceConfiguration build() {
             return new AuthServiceConfiguration(this);
```

Comment:
If the `withMinTimeBetweenJwksRequests` is not called, minTimeBetweenJwksRequests will be 0 and the AuthServiceConfiguration will be built with a '0 min between jwks requests'.
I would suggest to make the default here in the builder, and make the minTimeBetweenJwksRequests final in the  AuthServiceConfiguration class